guidelines/index.html

```diff
@@ -596,6 +596,14 @@
 					<p>When an authenticated session expires, the user can continue the activity without loss of data after re-authenticating.</p>
 				</section>
 			</section>
+      
+      <section class="sc">
+					<h4>Animation from interactions</h4>
+					<p class="conformance-level">A</p>
+					<p class="change">Unchanged</p>
+					<p>For significant animation triggered by a user action that is not an essential part of the action, there is a mechanism for the user to pause, stop or hide the animation whilst still performing the same action.</p>
```

Comment:
Hi @joshueoconnor ,

Thank you for your comment on the [February 7 2017 survey](https://www.w3.org/2002/09/wbs/35422/SC_20170207/results#xq4), Josh. You indicated "This SC needs more discussion" and wrote:

> Dont think you need the term significant. And I'm not sure how this is that different from WCAG 2.0 and pause/stop etc moving content??

As stated in my [January 17 status report to the working group](http://www.d.umn.edu/~lcarlson/wcagwg/animation/report.html) the SC is about extending 2.2.2 "Pause, stop hide" to cover an additional scenario: when a person unexpectedly initiates an animation such as in parallax scrolling. Parallax scrolling involves a background moving at a different speed than foreground content, creating a special effect as a person scrolls down a page. It can trigger nausea and migraines in people with vestibular disorders.  

The main area of refinement has been testability (size and duration) for incorporating into the "significant animation" definition. 

Does that help?